main.py

```diff
@@ -333,7 +336,7 @@ def prep_content(stats: dict[str, Any], language_count: int = 5, stop_at_other:
             + f"{lang_ratio:.2f}".zfill(5)
             + " %\n"
         )
-        if language_count == -1:
+        if wk_i.language_count == -1:
```

Comment:
This must be `language_count` instead of `wk_i.language_count`

```diff
- if wk_i.language_count == -1:
+ if language_count == -1:
```

Because in line 311

```py
language_count, stop_at_other = int(wk_i.language_count), bool(wk_i.stop_at_other)
```